src/data_traits.rs

```diff
@@ -238,6 +247,20 @@ unsafe impl<A> Data for OwnedArcRepr<A> {
             strides: self_.strides,
         }
     }
+
+    fn to_share<D>(self_: &ArrayBase<Self, D>) -> ArrayBase<OwnedArcRepr<Self::Elem>, D>
+    where
+        Self::Elem: Clone,
+        D: Dimension,
+    {
+        // to shared using clone of OwnedArcRepr without clone of raw data.
+        ArrayBase {
```

Comment:
Can't this method body just be `self_.clone()`. We should avoid writing out the ArrayBase constructor just to keep maintainability.

Reply:
That's right. Thanks for pointing that out.